app/models/concerns/hud_reports/households.rb

```diff
@@ -115,18 +115,16 @@ def hoh_age(household_id, date)
     end
 
     private def calculate_move_in_date(hh_id, she)
-      return nil unless she.move_in_date.present?
-
       move_in_date = she.move_in_date
       # If the move-in-date is valid, just use it
-      return move_in_date if move_in_date >= she.first_date_in_program
+      return move_in_date if move_in_date.present? && move_in_date >= she.first_date_in_program
 
       # If the client moved in before the entry date, and the HoH was present on the move-in date, use the
       # entry date as the move-in date.
       household_members = households[hh_id]
       hoh = household_members.detect { |hm| hm[:relationship_to_hoh] == 1 }
       return nil unless hoh.present?
-      return she.first_date_in_program if hoh[:entry_date] <= move_in_date
+      return she.first_date_in_program if hoh[:move_in_date].present? && hoh[:entry_date] <= (move_in_date || hoh[:move_in_date])
```

Comment:
I think this isn't quite right. Maybe:
```suggestion
      # Household was moved in when client arrived
      return she.first_date_in_program if hoh[:move_in_date].present? && hoh[:move_in_date] <= she.first_date_in_program
```
We might also want to short circuit if the HoH move-in date is < the HoH entry date, but that's more of a data quality issue.

Reply:
rerunning the test kits with this change and will add it in if they pass

Reply:
I think I've got something working for this. It's passing the test kit for the organization that was failing earlier. Running a full test kit to verify it doesn't break elsewhere and if we're good, I'll get it into the PR.

Reply:
@eanders a new commit for this has just been pushed to the PR